corehq/apps/app_manager/util.py

```diff
@@ -119,37 +119,34 @@ def split_path(path):
 
 
 def save_xform(app, form, xml):
```

Comment:
I think this function should never be called on `ShadowForm`s. For instance, the line
`form.source = xml`
will error because in shadow forms `form.source` is an object `@property`, not a `StringProperty`.